staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -1834,9 +1834,10 @@ type CSIPersistentVolumeSource struct {
 	// nodeExpandSecretRef is a reference to the secret object containing
 	// sensitive information to pass to the CSI driver to complete the CSI
 	// NodeExpandVolume call.
-	// This is an alpha field and requires enabling CSINodeExpandSecret feature gate.
+	// This is a beta field which is enabled default by CSINodeExpandSecret feature gate.
```

Comment:
Also update the internal types.go: https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/types.go#L1752

Reply:
done.. thanks!